src/TesApi.Tests/Repository/TesTaskPostgreSqlRepositoryIntegrationTests.cs

```diff
@@ -142,14 +146,20 @@ public async Task Create1mAndQuery1mAsync()
             sw.Restart();
             var runningTasks = await repository.GetItemsAsync(c => c.State == Models.TesState.RUNNINGEnum);
 
-            // Ensure performance is decent.  In manual testing on fast internet, this takes less than 5s typically
-            Assert.IsTrue(sw.Elapsed.TotalSeconds < 10);
+            // Ensure performance is decent
+            Assert.IsTrue(sw.Elapsed.TotalSeconds < 20);
             Console.WriteLine($"Retrieved {runningTasks.Count()} in {sw.Elapsed.TotalSeconds:n1}s");
             sw.Restart();
             var allOtherTasks = await repository.GetItemsAsync(c => c.State != Models.TesState.RUNNINGEnum);
             Console.WriteLine($"Retrieved {allOtherTasks.Count()} in {sw.Elapsed.TotalSeconds:n1}s");
             Console.WriteLine($"Total running tasks: {runningTasks.Count()}");
             Console.WriteLine($"Total other tasks: {allOtherTasks.Count()}");
+            var distinctRunningTasksIds = runningTasks.Select(i => i.Id).Distinct().Count();
```

Comment:
what I'd recommend is a single line "Assert.AreEqual(itemCount, runningTasks.Concat(allOtherTasks).Select(i => i.Id).Distinct().Count());"